proto/beacon/p2p/v1/types.proto

```diff
@@ -0,0 +1,128 @@
+
+syntax = "proto3";
+
+package ethereum.beacon.p2p.v1;
+
+import "google/protobuf/timestamp.proto";
+
+message BeaconState {
+  uint64 validator_set_change_slot = 1;
+  repeated ValidatorRecord validators = 2;
+  repeated CrosslinkRecord crosslinks = 3;
+  uint64 last_state_recalculation_slot = 4;
+  uint64 last_finalized_slot = 5;
+  uint64 last_justified_slot = 6;
+  uint64 justified_streak = 7;
+  repeated ShardAndCommitteeArray shard_and_committees_for_slots = 8;
+  repeated uint64 persistent_committees = 9;
+  repeated ShardReassignmentRecord persistent_committee_reassignment = 10;
+  bytes next_shuffling_seed = 11;
+  repeated uint64 deposits_penalized_in_period = 12;
+  bytes validator_delta_hash_chain = 13;
+  uint64 current_exit_seq = 14;
+  google.protobuf.Timestamp genesis_time = 15;
```

Comment:
genesis time should probably be uint64 or we have to do the translation again before ssz